tests/behavior/transforms/pipelines_complete.toml

```diff
@@ -1,178 +1,178 @@
-[[transforms.processing.logs.pipelines.aws_s3_access_logs.transforms]]
-type = "remap"
-source = '''
-.http.url_details |= parse_user_agent(.http.url)
-'''
-
-# Processor 4: Date Remapper: Define date_access as the official timestamp of the log
-[[transforms.processing.logs.pipelines.aws_s3_access_logs.transforms]]
-type = "remap"
-source = '''
-schema_set_timestamp(.date_access)
-'''
-
-# Processor 5: Category Processor: Categorize status code
-[[transforms.processing.logs.pipelines.aws_s3_access_logs.transforms]]
-type = "remap"
-source = '''
-.http.status_category = switch http.status_code {
-when 200...299:
-    "OK"
-when 300...399:
-    "notice"
-when 400...499:
-    "warning"
-}
-'''
-
-# Processor 6: Status Remapper: Set the log status based on the status code value
-[[transforms.processing.logs.pipelines.aws_s3_access_logs.transforms]]
-type = "remap"
-source = '''
-schema_set_status(.http.status_category)
-'''
-
-[transforms.processing.logs.pipelines.sudo]
-name = "Sudo"
-filter.type = "datadog_search"
-filter.source = "source:s3"
-
-[[transforms.processing.logs.pipelines.sudo.transforms]]
-type = "remap"
-source = '''
-. |= parse_grok(
-    value: .message,
-    pattern: "%{_sudo_user} : %{_sudo_tty}( %{_sudo_pwd})? %{_sudo_run_as_user}( %{_sudo_group})?( %{_sudo_tsid})?( %{_sudo_env})? (%{_sudo_command})?; (%{_sudo_lang} )?(%{_sudo_lc_type} )?.*",
-    aliases: {
-        _sudo_user: "%{notSpace:system.user}",
-        _sudo_tty: "TTY=%{notSpace:system.tty} ;",
-        _sudo_pwd: "PWD=%{notSpace:system.pwd} ;",
-        _sudo_run_as_user: "USER=%{notSpace:system.run_as_user} ;",
-        _sudo_group: "GROUP=%{notSpace:system.run_as_group} ;",
-        _sudo_tsid: "TSID=%{notSpace:system.tsid} ;",
-        _sudo_env: "ENV=%{data:system.env}",
-        _sudo_command: "COMMAND=%{data:system.cmd}",
-        _sudo_lang: "LANG=%{notSpace:system.lang}",
-        _sudo_lc_type: "LC_CTYPE=%{notSpace:system.lc_type}"
-    }
-)
-'''
-
-[transforms.standard_attributes]
-type = "remap"
-source = '''
-.error.stack ?= .exc_info ?? .stack
-.http.method ?= .billing_audit_request_method
-'''
-
-[transforms.sample]
-type = "sample"
-inputs = [ "processing" ]
-rate = 10
-
-[transforms.log_to_metrics]
-type = "log_to_metric"
-inputs = [ "processing" ]
-
-[[transforms.log_to_metrics.metrics]]
-field = "duration"
-name = "duration_total"
-namespace = "service"
-type = "counter"
-
-[transforms.log_to_metrics.metrics.tags]
-host = "${HOSTNAME}"
-region = "us-east-1"
-status = "{{status}}"
-
-[sinks.aws_s3]
-type = "aws_s3"
-inputs = ["processing"]
-bucket = "my-bucket"
-key_prefix = "date=%F/"
-compression = "gzip"
-region = "us-east-1"
-encoding.codec = "text"
-
-[sinks.datadog_logs]
-type = "datadog_logs"
-inputs = [ "processing" ]
-default_api_key = "${DATADOG_API_KEY_ENV_VAR}"
-compression = "gzip"
-
-[sinks.datadog_metrics]
-type = "datadog_metrics"
-inputs = [ "processing" ]
-api_key = "${DATADOG_API_KEY_ENV_VAR}"
+# [sources.datadog_agent]
```

Comment:
Could we just delete this file instead of commenting it? We can always revive from `git` if needed.

Reply:
Yea, sure thing. I'd temporarily commented it out while waiting on the Slack thread.